### DISCUSSION TITLE
Is there a way to rename the already created folder via Manage content ?

### DISCUSSION THREAD

**Discussion by reshma67** [microsoft/PubSec-Info-Assistant#717] [Q&A] [answered]
I would like to rename the folder which was already created and uploaded 50 docs into the same folder

**Comment by dayland** [comment, marked as answer]
We do not have this feature in the UX at the moment. Changing folder names would require updating chunk metatdata JSON and search index entries. 

For tips and pointers on what has to be changed, there are new features in our v1.1.1 release that help to "migrate" from one RG to another. Those same steps would be required to rename folders after files were ingested. 

It would likely be easier to delete and reupload the files in general.
